### DISCUSSION TITLE
Was Vandenberg in 170 mode?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#78] [General]
Or, are they sitting on an in-house 180 extension to DtCyber?

I've been wondering about this for some time now.

170 mode is quite plausible.  If it's anything like Point Mugu was, some of that code could be quite old, with no reason to "upgrade".

If they were in native 180 mode, and they do have an in-house version of this to do that, it's Vandenberg, we may never know...

**Comment by kej715** [comment]
Yes, they use ordinary DtCyber and run in 170 mode with NOS 2.8.7.

**Reply by KC5VDJ** [reply]
Okay, now it makes sense.  VE would be a major undertaking.  But then, I am planning an interesting project, see Pi-DI.  Had to ask.  Thanks.
